core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts

```diff
@@ -97,6 +97,23 @@ export class DotExperimentsListComponent {
         });
     }
 
+    /**
+     * Go to view Experiment Report
+     * @param {DotExperiment} experiment
+     * @returns void
+     * @memberof DotExperimentsShellComponent
+     */
+    goToViewExperimentReport(experiment: DotExperiment) {
+        this.router.navigate(['/edit-page/experiments/reports/', experiment.id], {
+            queryParams: {
+                editPageTab: null,
+                variantName: null,
+                experimentId: null
+            },
+            queryParamsHandling: 'merge'
+        });
```

Comment:
This should be a link and not a click button, when an href change is a navigation which means a link.

Reply:
That method comes from an output of a child component(`DotExperimentsListTableComponent`).
`(goToReport)="goToViewExperimentReport($event)"`